### ISSUE TITLE
gnome-terminal -e is deprecated

### ISSUE THREAD

**Issue by vsht** [p-e-w/argos#52]
I'm running argos on Fedora 28 with Gnome 3.28.2.  In the journal I have noticed the following warning 
related to argos:

```
Option “-e” is deprecated and might be removed in a later version of gnome-terminal.
Use “-- ” to terminate the options and put the command line to execute after it.
```

Apparently this comes from 

https://github.com/p-e-w/argos/blob/af6f3591607407831cf5054359e50ff485939174/argos%40pew.worldwidemann.com/menuitem.js#L66

Indeed, if you open a terminal and try to evaluate
```
gnome-terminal -e "bash -c \"ls -al; exec bash\"
```

it will produce the aforementioned warning. The new correct form seems to be
```
gnome-terminal -- bash -c "ls -al; exec bash"
```
I tried to put something like
```
argv = ["gnome-terminal", "--", "bash -c " + GLib.shell_quote(activeLine.bash + "; exec bash")];
```
in menuitem.js but that didn't really work. However, I don't really know any javascript, so perhaps I've  just  committed an elementary mistake somewhere.

Cheers,
Vladyslav

**Comment by boppy** [comment]
You'd been quite close! ;-)

This would work:

```js
argv = ["gnome-terminal", "--", "bash", "-c", activeLine.bash + "; exec bash"];
```

I just test-cased it with some weird string like

```bash
echo "test | bash=\"echo 'single quoted'\\\"double quoted\\\"'or just a \\\"'; sleep 5\""
```
@p-e-w what did I miss? ;-))

Another slight enhancement on this would be to use the users default shell instead of hardcoded bash where possible:

```js
argv = ["gnome-terminal", "--", "bash", "-c", activeLine.bash + "; exec ${SHELL:=bash}"];
```

**Comment by p-e-w** [comment]
Fixed by @boppy in #69.
